core/build.gradle.kts

```diff
@@ -81,6 +81,7 @@ dependencies {
     implementation(libs.ktorValidation)
```

Comment:
Commit message: Is this correct "Nullable properties are [...] marked as required"? Should this be "Non-nullable properties" or "marked as optional"?